main/src/com/google/refine/commands/row/AddRowsCommand.java

```diff
@@ -58,37 +58,46 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             return;
         }
 
+        processRequest(request, response);
```

Comment:
Splitting 6 lines of code into two extracted methods which call each other actually makes this code harder to read, in my opinion, because of the nesting, the separation of code which is essentially a linear flow, and the interruption of the method signatures.